functions/models/user.js

```diff
@@ -0,0 +1,54 @@
+const admin = require('firebase-admin');
+// TODO: add consts for field names in the db
+admin.initializeApp();
+const db = admin.firestore();
+let usersCollection = db.collection('users');
+class User {
+  //TODO: add data validation to all properties
+    constructor(uid) {        
+        if (!uid) {
+          console.log("Must provide a user uid when creating a new user");
+          return;
+          // TODO: Add better error handling here
+        }
+        this.uid = uid;  // this corresponds to Outcome Tracker's "System Name ID"
+        this.email = '';
+        this.firstName = '';
+        this.lastName = '';
+        this.dateCreated = Date.now();//TODO: switch this to a more readable date format
+    }
+    printAllFieldsToConsole() {
+      console.log (
+        "uid: " + this.uid + "\n" +
+        "First name: " + this.firstName + "\n" +
+        "Last name: " + this.lastName + "\n" +
+        "email: " + this.email + "\n")
+    }
+    createNewUserInFirestore() {
```

Comment:
I think this work probably addresses task #157 on the [project board](https://github.com/codeforbtv/cvoeo-app/projects/1).  Would you review that issue and either update it to reflect any changes you feel still need to be made, or just tag it with this PR number (169) and move it to the completed column?